vendor/assets/javascripts/bootstrap-table-filter-extension.js

```diff
@@ -0,0 +1,70 @@
+

```

Comment:
This file is in the gem https://github.com/bjevanchiu/bootstrap-table-rails/blob/master/vendor/assets/javascripts/extensions/bootstrap-table-filter.js: You can require the gem version with `//= require 'extensions/bootstrap-table-filter'`. Only difference is `throw 'Error: searchCallback not defined'`, which shouldn't matter too much?